Tools/TestWebKitAPI/Tests/WebKitCocoa/ScreenTime.mm

```diff
@@ -121,6 +121,49 @@ - (void)observeValueForKeyPath:(NSString *)keyPath ofObject:(id)object change:(N
 }
 @end
 
+static BOOL blurredBlockingViewIsPresent(RetainPtr<TestWKWebView> webView)
+{
+#if PLATFORM(IOS_FAMILY)
+    for (UIView *subview in [webView subviews]) {
+        if ([subview isKindOfClass:[UIVisualEffectView class]])
+            return true;
+    }
+#else
+    for (NSView *subview in [webView subviews]) {
+        if ([subview isKindOfClass:[NSVisualEffectView class]])
+            return true;
+    }
+#endif
+    return false;
+}
+
+static void testSuppressScreenTimeBlocking(bool suppressScreenTimeBlocking, bool isRemoveTest = false)
```

Comment:
Optional: Boolean trap. Makes the call sites hard to follow. https://webkit.org/code-style-guidelines/#names-enum-to-bool

```suggestion
enum class SuppressScreenTimeBlocking : bool { No, Yes };
enum class TestBlockingViewRemoval : bool { No, Yes };

static void testSuppressScreenTimeBlocking(SuppressScreenTimeBlocking suppressScreenTimeBlocking, TestBlockingViewRemoval testBlockingViewRemoval = TestBlockingViewRemoval::No)
```